RNA-seq/03-gastric_cancer_exploratory.Rmd

```diff
@@ -174,12 +174,17 @@ Let's use ggplot to visualize the first two principal components.
 
 ```{r color-by-batch, live = TRUE}
 # Color points by "batch" and use shape to indicate the tissue of origin
-ggplot2::ggplot(pca_data, ggplot2::aes(PC1, PC2,
-                                       color = batch,
-                                       shape = tissue)) +
+ggplot2::ggplot(pca_data,
+  ggplot2::aes(
+    x = PC1, 
+    y = PC2,
+    color = batch,
+    shape = tissue
+  )
+) +
   ggplot2::geom_point(size = 3) +
-  ggplot2::xlab(paste0("PC1: ", percent_var[1],"% variance")) +
-  ggplot2::ylab(paste0("PC2: ", percent_var[2],"% variance")) +
+  ggplot2::xlab(paste0("PC1: ", percent_var[1], "% variance")) +
+  ggplot2::ylab(paste0("PC2: ", percent_var[2], "% variance")) +
```

Comment:
We should update this to `ggplot2::labs()` (and this should apply to other plots in the module, too). Looks like these are the only `ggplot` labs lines in the module too 👍

Reply:
There are other places we use `labs()` so I was kind of okay with showing both.

Reply:
🤷‍♀️ 

```
(base) RNA-seq ➤ grep "ylab" *Rmd 
03-gastric_cancer_exploratory.Rmd:  ggplot2::ylab(paste0("PC2: ", percent_var[2], "% variance")) +
```